lightning/src/ln/channelmanager.rs

```diff
@@ -4049,6 +4072,33 @@ impl<Signer: Sign, M: Deref, T: Deref, K: Deref, F: Deref, L: Deref> MessageSend
 				result = NotifyOption::DoPersist;
 			}
 
+			if self.startup_complete.load(Ordering::Relaxed) != 0 {
```

Comment:
Seems a little odd to do this in `get_and_clear_pending_msg_events`. Would it be any different if the call to `timer_tick_occurred` in the background processor was called unconditionally before the loop?

Reply:
That would work too for this specific case, though I don't think we should be super trigger-happy with the BroadcastChannelUpdate stuff on startup as we should give some grace time for peers to reconnect before we spam the network with updates. In general, I think it wont be uncommon to have a user on mobile who regularly starts and stops again within one minute, never hitting the timer at all.

Reply:
Done in #985.